src/chrome/content/rules/FAZ.xml

```diff
@@ -12,13 +12,13 @@
 
   <rule from="^http://fazarchive\.faz\.net/(content|css|images)/" to="https://fazarchive.faz.net/$1/"/>
 
-  <rule from="^http://(blogs|charts|faz-community|gets|kfz-versicherung|media0|media1|boersenlexikon|ts|verlag2)\.faz\.net/" to="https://$2.faz.net/"/>
+  <rule from="^http://(blogs|charts|faz-community|gets|kfz-versicherung|media0|media1|boersenlexikon|ts|verlag2)\.faz\.net/" to="https://$1.faz.net/"/>
 
-  <rule from="^http://(verlag\.|www\.)?faz\.(de|net)/(2\.5\.7/|cacheproxy300|f30|favicon\.ico|img/|l\.gif|polopoly_fs/)" to="https://www.faz.net/$3"/>
+  <rule from="^http://(verlag\.|www\.)?faz\.(de|net)/(2\.5\.7/|cacheproxy300|f30|favicon\.ico|img/|l\.gif|polopoly_fs/)" to="https://www.faz.net/$1"/>
 
-  <rule from="^http://(www\.)?faz(job\.net|-institute\.de)/" to="https://faz$4/"/>
+  <rule from="^http://(www\.)?faz(job\.net|-institute\.de)/" to="https://faz$1/"/>
```

Comment:
Remove this line completely.

The domain `faz-institute.de` is not registered.

https://fazjob.net/ and https://www.fazjob.net/ redirect to http.